Tests/BrowserServicesKitTests/ContentBlocker/SurrogatesUserScriptTests.swift

```diff
@@ -80,6 +80,14 @@ class SurrogatesUserScriptsTests: XCTestCase {
 """
 
     static let exampleSurrogates = """
+        # This file contains "surrogates". Surrogates are small scripts that our apps and extensions serve in place of trackers that cause site breakage when blocked.
+        # Learn more: https://github.com/duckduckgo/tracker-surrogates
+        badtracker.net/track.js application/javascript
+        (() => {
+            'use strict';
+            window.bad = 1;
+        })();
+
```

Comment:
I added some other sample trackers to make it match the real format. We're already checking the surrogate function is called so I think this is pretty robust for checking this isn't broken.